tools/cldr-apps/js/src/views/AboutPanel.vue

```diff
@@ -1,73 +1,70 @@
 <template>
-    <div id="home">
-        <p class='hang'>
```

Comment:
When I make a real (not draft) pr, I can back off on the changes that are only .vue formatting by npx prettier -- let's coordinate on that when convenient

Reply:
or just make a separate PR that's just the formatting?